e2e-tests/cypress/tests/integration/channels/files_and_attachments/file_preview_image_spec.js

```diff
@@ -103,6 +103,18 @@ describe('Upload Files - Image', () => {
 
         testImage(properties);
     });
+
+    it('MM-T2264_7 - WEBP', () => {
```

Comment:
In a separate PR, could you please update the test case at https://github.com/mattermost/mattermost-test-management/blob/main/data/test-cases/channels/files-and-attachments/MM-T2264.md to include step 7 for posting a webp image?

Reply:
@saturninoabril 

Before creating new Pull Request, I'd like to confirm following things:

1. In new PR, I have to make branch from current master branch. Right?
2. In new PR that contains changes for that Markdown, should I add test spec file and image file too?
3. In branch name and PR name, I think these two numbers are eligible. Which should I use PR's Number (25784) or Original Ticket number (11777).

Thanks

Reply:
1. No. Should branch off the `main` branch.
2. No need to add the test spec file nor the image file. Just a generic test instruction referencing a WEBP file.
3. Should use ticket number `MM-11777` then you may add link to this PR in the description.

Reply:
@saturninoabril 

I made PR in that repository.

https://github.com/mattermost/mattermost-test-management/pull/150